oneandone/src/test/java/org/apache/jclouds/oneandone/rest/internal/BaseOneAndOneLiveTest.java

```diff
@@ -102,6 +105,16 @@ protected Server createServer(String serverName) {
               .powerOn(Boolean.TRUE).build());
    }
 
+   protected Server updateServerStatus(Server server) {
```

Comment:
That's fine, my concern is to have a method that encapsulates that "sleep". Is this need to wait a common pattern users will have to deal with? If the answer is yes, we should consider providing a predicate users can easily use to wait until a server is ready / in the desired state.